src/backup-metadata/config/cfmetadata.yml

```diff
@@ -27,14 +27,14 @@ spec:
         name: cf-metadata
         command: ["/cnb/process/wait"]
         env:
-        - name: CF_API
```

Comment:
**Note:** This file will go away soon once we colocate the container within the `cf-api-server` deployment. This PR is just a necessary step toward that.